### PR TITLE
Update parent pom and sonar-analyzer-commons

### PR DESCRIPTION
**Pull request by andrei-epure-sonarsource**

### CODE REVIEW

File: sonar-csharp-plugin/pom.xml [hunk 1]
```diff
@@ -40,30 +40,34 @@
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
     <sonarAnalyzer.workDirectory>${project.build.directory}/analyzer</sonarAnalyzer.workDirectory>
     <documentationDirectory>${project.basedir}/src/main/resources</documentationDirectory>
+    <sonar.analyzer.commons>1.14.1.690</sonar.analyzer.commons>
```
Comment (pavel-mikula-sonarsource) [inline sonar-csharp-plugin/pom.xml:43]: Does this have to be on 3 different places? Can we define the property in the root pom once?

File: sonar-dotnet-shared-library/pom.xml [hunk 2]
```diff
@@ -20,34 +20,47 @@
...
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
       <version>${protobuf.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.sonarsource.analyzer-commons</groupId>
+      <artifactId>sonar-analyzer-commons</artifactId>
+      <version>${sonar.analyzer.commons}</version>
+      <scope>compile</scope>
```
Comment (pavel-mikula-sonarsource) [inline sonar-dotnet-shared-library/pom.xml:56]: _For consistency:_ The other occurrences of compiled dependencies do not use explicit value (like the one just above this, or the modified ones in other two POMs)
```suggestion
```

File: sonar-dotnet-shared-library/pom.xml [hunk 3]
```diff
@@ -20,34 +20,47 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.sonarsource.analyzer-commons</groupId>
+      <artifactId>sonar-xml-parsing</artifactId>
+      <version>${sonar.analyzer.commons}</version>
+      <scope>compile</scope>
```
Comment (pavel-mikula-sonarsource) [inline sonar-dotnet-shared-library/pom.xml:62]: Same
```suggestion
```

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM, my comments/questions are optional and we can do them later (if we can do them)

File: pom.xml [hunk 4]
```diff
@@ -54,6 +54,8 @@
     <artifactsToDownload>${project.groupId}:SonarAnalyzer.CSharp:nupkg,${project.groupId}:SonarAnalyzer.VisualBasic:nupkg</artifactsToDownload>
     <!-- We are ignoring java doc warnings - this is because we are using JDK 11. Ideally we should not do that. -->
     <doclint>none</doclint>
+    <sonar.analyzer.commons>1.14.1.690</sonar.analyzer.commons>
+    <sonar.version>7.9</sonar.version>
```
Comment (andrei-epure-sonarsource) [inline pom.xml:58]: note: I tried with 8.x (latest and even 8.0) and they removed the `import org.sonar.api.internal.google.common.annotations.VisibleForTesting;` API

I will ask if it's a problem at this point or we can ship it.

Comment (pavel-mikula-sonarsource) [reply pom.xml:58]: We have 
```
    <sonar.version>7.9</sonar.version>
    <sonarLTS.version>7.9</sonarLTS.version>
```
I think we should update them both after LTS release

Comment (pavel-mikula-sonarsource) [reply pom.xml:58]: Why not to move `sonarLTS.version` as well when moving `sonar.version`? They were next to each other

Comment (andrei-epure-sonarsource) [reply pom.xml:58]: sonarLTS isn't in sonar-dotnet-shared-library, so I didn't move it in the parent.

**Review by pavel-mikula-sonarsource — APPROVED** [review]
Still LGTM

**Pull request merged**